index.html

```diff
@@ -6,12 +6,32 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <script src="index.js" defer></script>
-  <link rel="stylesheet" href="style.css">
+  <script src="./index.js" defer></script>
+  <link rel="stylesheet" href="./style.css">
   <title>Awesome Books</title>
 </head>
 
 <body>
+
+  <header>
+    <nav>
+      <div>Awesome Books</div>
+      <button type="button" class="burger-menu" id="burger-menu" data-toggled="false">
+        <div class="line line1"></div>
+        <div class="line line2"></div>
+        <div class="line line3"></div>
+      </button>
+      <ul class="nav-menu">
+        <li class="nav-item">List</li>
+        <li class="nav-item">Add new</li>
```

Comment:
- [ ] [OPTIONAL] Good job so far 👍 In my opinion, your add and list section appear together when you refresh the web page, the rest is fine 😃 

![image](https://user-images.githubusercontent.com/63915024/198056956-fde43afa-63dd-4f82-a387-83fcb3911e16.png)